cmd/template/cluster/provider/templates/capa/types.go

```diff
@@ -0,0 +1,53 @@
+package capa
+
+type ClusterConfig struct {
```

Comment:
I feel like we could structure this better so the common values are in one package and the provider-specific values are in the provider packages. Fine as a start though.